Message user, not channel

kawa-kokosowa opened the pull request:
The API isn't clear, but apparently `rtm_send_message()` can be
used for both sending messages to channels _and_ users. I have
personally not tested this, I just presume it works.

Please test this. This branch/commit addresses issue #2.

Commit:
Message user, not channel

The API isn't clear, but apparently `rtm_send_message()` can be
used for both sending messages to channels *and* users. I have
personally not tested this, I just presume it works.

Please test this. This branch/commit addresses issue #2.

Commit:
Changed to slacker. A lot better.

rein commented:
Pushed version which works, just needs adapting to use greeting.txt and perhaps slight cleanup.

lillian-lemmer commented:
Thanks :+1: will wrap up tonight.

Commit:
Cleanup, apikey goes in ini, add requirements file

You can use `bot_config.ini` to set the Slack bot API key.

Add `requirements.txt` for installing requirements with `pip`.

Update `README.md` to reflect changes.

lillian-lemmer commented:
Commit 797dffc is untested.

Commit:
add [bot] to ini

rein commented:
Ok. Have tested with the newest commit (8af802a), all works!
